packages/devextreme/scss/widgets/fluent/tooltip/_colors.scss

```diff
@@ -20,17 +20,14 @@ $tooltip-bg: null !default;
 
 $tooltip-shadow-color: null !default;
 $tooltip-border-radius: $base-border-radius !default;
-$overlay-content-shadow-color: null !default;
 
 @if $mode == "light" {
   $tooltip-color: $base-text-color !default;
   $tooltip-bg: $base-bg !default;
-  $tooltip-shadow-color: 0 0 2px 0 rgba(0, 0, 0, 0.12), 0 4px 8px 0 rgba(0, 0, 0, 0.14) !default;
 }
 
 @if $mode == "dark" {
   $tooltip-color: $base-text-color !default;
   $tooltip-bg: $base-neutral-background1-hover !default;
-  $tooltip-shadow-color: 0 0 2px 0 rgba(0, 0, 0, 0.24), 0 4px 8px 0 rgba(0, 0, 0, 0.28) !default;
```

Comment:
we deleted this but it's used in `$tooltip-shadow-color`
moreover, declaration above still exists